js/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -0,0 +1,123 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var isUndefined = require( "lodash/isUndefined" );
+
+var MissingArgument = require( "../../js/errors/missingArgument" );
+/**
+ * @param {object} app The app
+ * @param {object} args An arguments object with usedKeywords, searchUrl, postUrl,
+ * @param {object} args.usedKeywords An object with keywords and ids where they are used.
+ * @param {string} args.searchUrl The url used to link to a search page when multiple usages of the keyword are found.
+ * @param {string} args.postUrl The url used to link to a post when 1 usage of the keyword is found.
+ * @param {Assessor} args.assessor
+ * @param {object} i18n The i18n object used for translations
+ * @constructor
+ */
+var PreviouslyUsedKeyword = function( app, args, i18n ) {
+	if ( isUndefined( app ) ) {
+		throw new MissingArgument( "The previously keyword plugin requires the YoastSEO app" );
+	}
+	if ( isUndefined( args ) ) {
```

Comment:
I would move this below the `i18n` if. Because the `app` and `i18n` are required.